storage-inventory/ops/README.md

```diff
@@ -10,76 +10,101 @@ Read the entire document or jump to a relevant section below.
     * [System Architecture](#system-architecture)
-    * [Basic Standalone Storage Site](#basic-standalone-storage-site)
-    * [Supporting Infrastructure and Services](#supporting-infrastructure-and-services)
-    * [Client Software](#client-software)
-3. [Deployment Prerequisites](#deployment-prerequisites)
+    * [Standalone Storage Site](#standalone-storage-site)
+1. [Client Software](#client-software)
+1. [Deployment Prerequisites](#deployment-prerequisites)
     * [Hardware Requirements](#hardware-requirements)
     * [Software Requirements](#software-requirements)
-4. [Deployment Steps](#deployment-steps)
-5. [Configuration and Customization](#configuration-and-customization)
-    * [Web Services Configuration](#web-services-configuration)
-    * [Application Configuration](#application-configuration)
-    * [A word about credentials](#a-word-about-credentials)
-6. [Management and Monitoring](#management-and-monitoring)    
-7. [Maintenance and Upgrades](#maintenance-and-upgrades)
-8. [Conclusion](#conclusion)
-    * [Deployment Scenarios](#deployment-scenarios)
-    * [Troubleshooting Guide](#troubleshooting-guide)
+1. [Deployment](#deployment)
+1. [Healthchecks and Monitoring](#healthchecks-and-monitoring)    
+1. [FAQ](#faq)
 
 
 ## Introduction
 
 ### Key Storage Inventory Terms and Concepts
 
-- _Artifact_: a representation of a file and its metadata in SI. File and artifact are used interchangebly through the document
-- _resourceID/serviceID_: Unique ID in a URI form associated with a deployed service. Examples: `ivo://opencadc.org/minoc`
-- _bucket_: unit of work for SI applications. More details TBD.
+- _Artifact_: a representation of a file and its metadata in SI. The terms _file_ and _artifact_ are often used interchangeably through the document.
+- _bucket_: Analogous to an S3 bucket or a folder, but specifically referring to the Storage Inventory implmentation. Buckets are created on storage (on a POSIX file-system, these would be just directories) and artifacts are distributed among these.  SI applications can be configured to work on subsets of buckets. 
+- _collection_: A _collection_ is a set of artifacts/files/observations that form a logical group in your system.  For example, for a multi-mission data centre like the CADC, collections are often the set of all data from a telescope.  A collection could also be all artifacts/files that compose a special data release for a single telescope (e.g. 'DR1').  The terms _collection_ and _namespace_ are often used interchangeably, but _namespace_ has a key role in defining artifact metadata.
+- _namespace_: A _namespace_ is the SI identifier for a _collection_ -- this namespace becomes part of artifact's unique metadata: the URI for an artifact is '_namespace/filepath_'.  A namespace can be any unique name in SI (**Pat - any restrictions?**), e.g. 'ska:askap', 'cadc:CFHT'.  The _filepath_ could be just a file name (e.g. test1.fits) or a relative path (e.g. proc1/test1.fits).
```

Comment:
Artifact.uri are URIs of the form {scheme}:{scheme-specific-part} where {scheme-specific-part} is a relative path and the last path component is (nominally) the file name. I don't really see how separating collection and filepath helps that much.... that's really just how we (CADC) think about this.

There are several uses for URI naming patterns (not just permissions) so it is important to think about before putting a bunch of files. Namespace is a concept that (as currently implemented ) means "a URI prefix that ends with a separator (: or /)".

Reply:
Are ':' and '/' treated the same in parsing a namespace?  So, instead of having `cadc:CFHT/` as a namespace, we could use `cadc/CFHT`?  I'm not suggesting anyone use this, I just want to be sure.